GetBlogFeeds.cs

```diff
@@ -40,95 +42,116 @@ protected override async Task<IEnumerable<IDocument>> ExecuteInputAsync(IDocumen
+                                                    || string.Equals(content.Name, "subtitle", StringComparison.OrdinalIgnoreCase))
+                                                {
+                                                    description = content.Value;
+                                                }
+                                                break;
                                         }
                                     }
+                                    catch (Exception ex)
+                                    {
+                                        context.LogWarning($"Exception while processing {feedReader.ElementType} in {feed}: {ex.Message}");
+                                    }
                                 }
                             }
                         }
-                    }
 
-                    // Get a new document with feed metadata
-                    MetadataItems metadata = new MetadataItems();
-                    if (items.Count > 0)
-                    {
-                        BlogFeedItem[] feedItems = items
-                            .Select(x => new BlogFeedItem(x, _recent, website))
-                            .OrderByDescending(x => x.Published)
-                            .Take(50) // Only take the 50 most recent items
-                            .ToArray();
-                        metadata.Add(SiteKeys.BlogFeedItems, feedItems);
-                        return input.Clone(metadata).Yield();
+                        // Get a new document with feed metadata
+                        MetadataItems metadata = new MetadataItems();
+                        if (items.Count > 0)
+                        {
+                            BlogFeedItem[] feedItems = items
+                                .Select(x => new BlogFeedItem(x, _recent, website))
+                                .OrderByDescending(x => x.Published)
+                                .Take(50) // Only take the 50 most recent items
+                                .ToArray();
+                            metadata.Add(SiteKeys.BlogFeedItems, feedItems);
+                            return input.Clone(metadata).Yield();
+                        }
                     }
                 }
                 catch (Exception ex)
                 {
                     context.LogWarning($"Error getting feed for {feed}: {ex.Message}");
                 }
+            } 
+            else 
+            {
+              var speaker = input.GetString("Title");
+              if (!string.IsNullOrWhiteSpace(speaker))
+              {
+                context.LogWarning($"No RSS specified for {speaker}.");
+              }
+              else
+              {
+                context.LogWarning($"No RSS specified for unknown speaker.");
+              }
             }
-
+            
             return null;
```

Comment:
In case rebasing on #392 isn't clean, the only other thing that needs to change is this line. Should be `return input.Yield()` instead of `return null`.